apps/vendor.js

```diff
@@ -5,29 +5,38 @@ var faker = require('faker');
 let storeName = process.env.STORE_NAME;
 const emitter = require('../lib/events.js');
 
-emitter.on('delivered', deliveryHandler);
 
 const randomStore = `${faker.phone.phoneNumber()}`;
 const randomOrderID = `${faker.random.number()}`;
 const randomOrderName = `${faker.name.findName()}`;
 const randomAddress = `${faker.address.streetAddress()}`;
 
+emitter.on('pickup', orderHandler);
+emitter.on('delivered', deliveryHandler);
 
 
-function deliveryHandler() {
-  setInterval(order, () => {
+function orderHandler() {
+  setInterval('pickup', () => {
     let delivery = {
       store: randomStore,
       orderID: randomOrderID,
       customer: randomOrderName, 
       address: randomAddress,
     };
+    // Need to figure out how to create new delivery object below
+    // .create(delivery);
     emitter.emit('pickup', delivery);
-    console.log(order);
   }, 5000); 
   
-};
+}
 
+function deliveryHandler(order) {
+  console.log(`Thank you for delivering ${order.orderID}`);
```

Comment:
With your test for this function, you are getting this message back. But when you look at your test, you added more to this line.